pyproject.toml

```diff
@@ -94,9 +123,23 @@ filterwarnings = [
     "ignore:You are encoding a BooleanParameterType:UserWarning"
 ]
 
+# Note: Ruff uses the "nearest" configuration file at runtime so configurations may be overridden
+# e.g. see ruff.toml in the tests directory
```

Comment:
The ruff.toml looks like basically the same setup except for the "S" selection. What about adding a `per-file-ignores` clause here instead of a separate `ruff.toml` to try and keep everything in one place configuration-wise?

Reply:
Oooh nice. I didn't know about that option. Way nicer.